CHANGELOG.rst

```diff
@@ -6,22 +6,93 @@ DrizzlePac Release Notes
 
 The version of DrizzlePac can be identified using ::
 
-> python
->>> import drizzlepac
->>> drizzlepac.__version__
-
+  > python
+  >>> import drizzlepac
+  >>> drizzlepac.__version__
 
 The following notes provide some details on what has been revised for each
 version in reverse chronological order (most recent version at the top
-of the list).
+of the list).  Numbers at the end of each item is the Github Pull Request (PR)
+number of the code change for that issue.  These PR's can be viewed at:
+
+    https://github.com/spacetelescope/drizzlepac/pulls
 
-3.3.2 (unreleased)
+
+3.4.0 (unreleased)
 ==================
+This major release adds support for multi-visit mosaic (MVM) processing, in
+addition to including numerous revisions to try to align more datasets
+successfully to GAIA during pipeline and (single-visit mosaic) SVM processing.
+Multi-visit mosaics introduce the concept of SkyCells with new code added to define
```

Comment:
Line 26 
Multi-visit mosaics ==> MVMs

Reply:
Fixed in 99a595e.

Reply:
This commit also adds a separate (admittedly terse) definition of the SkyCell.